.github/workflows/windows-build.yml

```diff
@@ -2,7 +2,10 @@ on:
   push:
     branches:
       - master
-  pull_request:
+      - /feature.*/
+    tags:
+      - v1.*
+      - v2.*
```

Comment:
In Travis we do this for `tag IS present` - any reason to be different here?

Reply:
we can loosen the restrictions here if you would rather